<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Light Novel Exporter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="wdgList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>691</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblType">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblLanguage">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>500</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblGenres">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>550</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imgCover">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>251</width>
      <height>331</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowList">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>670</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBooks">
    <property name="title">
     <string>Novel List</string>
    </property>
    <addaction name="actAddBook"/>
    <addaction name="actAddBookURL"/>
    <addaction name="actRemoveBook"/>
   </widget>
   <widget class="QMenu" name="menuNovel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Novel</string>
    </property>
    <addaction name="actDownload"/>
    <addaction name="actGenerate"/>
   </widget>
   <addaction name="menuBooks"/>
   <addaction name="menuNovel"/>
  </widget>
  <action name="actAddBookURL">
   <property name="text">
    <string>Add from URL</string>
   </property>
  </action>
  <action name="actDownload">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Download Chapter/s</string>
   </property>
  </action>
  <action name="actGenerate">
   <property name="text">
    <string>Generate ePUB</string>
   </property>
  </action>
  <action name="actRemoveBook">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove Book</string>
   </property>
  </action>
  <action name="actAddBook">
   <property name="text">
    <string>Add Book</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
